spec/integration/rest/authy/v1/service.spec.js

```diff
@@ -50,6 +53,25 @@ describe('Service', function() {
           url: url,
           data: values
       }));
+
+      var headers = {
+        'Twilio-Authy-Sandbox-Mode': 'twilio_authy_sandbox_mode',
+        'Accept': 'application/json',
+        'Accept-Charset': 'utf-8'
+      };
+      headers['User-Agent'] = util.format(
+        'twilio-node/%s (node.js %s)',
+        moduleInfo.version,
+        process.version
+      );
+      if (!headers['Content-Type']) {
+        headers['Content-Type'] = 'application/x-www-form-urlencoded';
+      }
+      holodeck.assertHasRequest(new Request({
+        method: 'POST',
+        url: url,
+        headers: headers
+      }));
```

Comment:
Should create a util somewhere that does most of this since it's repeated a lot.

Reply:
Moved this logic to Holodeck. `holodeck.assertHasRequest` now checks if there's a match when standard headers are appended.